### ISSUE TITLE
Admin order page crashed when filtering on nothing.

### ISSUE THREAD

**Issue by AaronTajriba** [myparcelnl/prestashop#133]
### Module version

1.5.1

### PrestaShop version

1.7.7.8

### PHP version

7.3

### What went wrong?

De order overview page crashes when is filtered on something that does not exist
.
![image0](https://user-images.githubusercontent.com/70962249/160349598-0e1b7647-930d-4573-9896-0cd7c0a4d5eb.jpeg)

### Reproduction steps

1. Navigate to the orders page
2. filter on someting that does not exist in your orderslist (examle name test does not exitst)
3. See the error what i got

### Relevant log output

```shell
none
```

### Additional context

I see this as a Major issue witch needs to get fixed. We are not able to work properly in the backoffice. 

We need to clear the ps_admin_filter table before we can do anything.

**Comment by TomanDesign** [comment]
The same situation, also we get the same error when trying to create a new order in the admin panel. 
Amin address: ..../index.php/sell/orders/new?_token=7a5Rz4iG99SEyLq_xR3JNGcSXPY4V9JNrOYlEXygXXs

**Module version**
1.5.1

**PrestaShop version**
1.7.7.1

**PHP version**
7.3

**Comment by mariuszsienkiewicz** [comment]
While waiting for the developers to fix this problem, I made the following changes to make it work:

in `src/Module/Hooks/RenderService.php`, right before class declaration I added SymfonyContainer:

```php
use PrestaShop\PrestaShop\Adapter\SymfonyContainer;

class RenderService
```

and in constructor of this class I have changed this:
```php
$this->twig    = $this->module->get('twig');
```

to this:
```php
$this->twig    = SymfonyContainer::getInstance()->get('twig');
```

After these changes order search works as expected.

**Comment by AaronTajriba** [comment]
@mariuszsienkiewicz can confirm, this works! Thank you.  Dev's are you looking with us?

**Comment by mariuszsienkiewicz** [comment]
Unfortunately, those developers are not the best in terms of communication (see #112) and support (**edit**: it is not as bad as I thought before I see that they implemented the fix here: 9b4b6269f8a0527140aaf05f57281b5fb90f21ee, so only communication right now is really bad - so I'm gonna remove some of my comments).  
@TomanDesign if you still have a problem with an order creation via the administrator panel try the fix above while we are waiting for a proper fix by devs.

**Comment by TomanDesign** [comment]
@mariuszsienkiewicz I have implemented the above solution, unfortunately the problem with an order creation still exists.

**Comment by wthijmen** [comment]
Hi @AaronTajriba,

This issue has been solved since the release of [v1.5.3](https://github.com/myparcelnl/prestashop/releases/tag/v1.5.3)
